contact-widgets.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Contact Widgets
  * Description: Beautifully display social media and contact information on your website with these simple widgets.
- * Version: 1.3.3
+ * Version: 1.3.4
```

Comment:
@jbardo I think the version was incremented because we tagged the previous release when we bumped the tested up to version to 4.7.